third_party/mdbook/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "cargo_raze_deps"
```

Comment:
Can we rename both of these packages please? We're not using Raze anymore and somebody was confused by this name today.

Maybe just `name = "cargo_mdbook"` and `name = "cargo_opentitan"`?